Source/JavaScriptCore/runtime/LiteralParser.h

```diff
@@ -180,6 +189,7 @@ class LiteralParser {
     JSValue tryLiteralParsePrimitiveValue()
     {
         ASSERT(m_mode == StrictJSON);
+        ASSERT(reviverMode == JSONReviverMode::Disabled);
```

Comment:
Ditto.  Make this a `static_assert` on `reviverMode`.

Reply:
Ditto.